### ISSUE TITLE
Hover editor does not pop-up upon clicking Ctrl and hovering over the link after recent Obsidian update

### ISSUE THREAD

**Issue by Sriramgireesh** [nothingislost/obsidian-hover-editor#182]
After the recent update to Obsidian 1.0.0, the Hover editor does not pop-up like it used to when I hover over the link and press Ctrl.

**Comment by pjeby** [comment]
What links, specifically?  It works fine for me with links on kanban boards and in markdown notes (both editing and reading), recent files, backlinks, etc.

Do you have the most recent version of Hover Editor installed?

**Comment by dredhorse** [comment]
works for me too. What does not work though it the jump to a header or blockref

**Comment by pjeby** [comment]
Jumping to headers works for me on 1.0 as well.

**Comment by DavidLiebeFreunde** [comment]
I am still having that issue. Page Preview of Obsidian itself is working with a hover of the mouse over an internal link in wiki-style and then pressing ctrl (in every view mode, edit, view and live-preview). But with Hover Editor turned on in the plugin settings the previews dont appear anymore, nomatter what view mode.
OS: Windows 10
Obsidian: 1.0.0 (Installer 0.14.6)

**Comment by pjeby** [comment]
Since a deinstall and reinstall fixed it, it is likely that the problem was in the plugin settings (assuming you had the latest version of Hover Editor).  If anyone is still having the problem, please check your settings, especially the setting for the delay before popping up.

**Comment by ZeliTheZealot** [comment]
Yes @pjeby is correct. I tried to set the Hover Trigger Delay too low (100ms is too low but 120ms is okay), which caused the preview editor to not appear.

**Comment by pjeby** [comment]
Closing this since it doesn't appear reproducible, and there is now a new version of Hover Editor (0.11.7) that can be tried for anyone still experiencing the problem.

**Comment by pjeby** [comment]
Ok, looks like having too low of a delay setting is the problem; I'll have to investigate adding a minimum or seeing if there's some other way to fix the issue.

In the meantime, anyone experiencing this issue should increase the delay until it works (apparently 120ms is a good place to start).

**Comment by Meyer17** [comment]
FYI for future users—I didn’t change the hover trigger delay from default when I initially downloaded it and had this issue, but it went away after I changed the delay setting to 400. Now, I can adjust it as low as 100 without any issues, so it appears that just changing the delay time at all fixed the issue for me.
